src/components/ExportKmlService.js

```diff
@@ -88,17 +88,17 @@ goog.require('ga_browsersniffer_service');
 
           if (exportFeatures.length > 0) {
             if (exportFeatures.length == 1) {
-              //force the add of a <Document> node
+              // force the add of a <Document> node
               exportFeatures.push(new ol.Feature());
             }
             kmlString = gaKml.getFormat().writeFeatures(exportFeatures);
             // Remove no image hack
-            kmlString = kmlString.
-                replace(/<Icon>\s*<href>noimage<\/href>\s*<\/Icon>/g, '');
+            kmlString = kmlString
+                .replace(/<Icon>\s*<href>noimage<\/href>\s*<\/Icon>/g, '');
```

Comment:
IMO, the older version is much more readably. It's clear that something will come in next line. With new version, it's not clear...